pyfarm/agent/http/api/state.py

```diff
@@ -109,7 +109,8 @@ def post(self, **kwargs):
                 logger.info("Created run control file %s",
                             config["run_control_file"])
 
-        if not config["jobtype"] or data.get("immediately", False):
+        if ("jobtypes" not in config or not config["jobtypes"] or
```

Comment:
Hm, no. I don't just want to catch when it is ```None```, I also need to check if it's an empty list.